spec/src/main/asciidoc/metrics_spec.adoc

```diff
@@ -126,6 +126,7 @@ Details of this Java API are described in <<app-metrics-api>>.
 
 Application specific metrics are exposed under `/metrics/application`.
 
+The application context root must be prepended to all application metrics to prevent collisions where multiple applications are deployed to the same server. Where the implementation does not support multiple applications on the same server, or an application is deployed to the root context (`/`), it is not necessary to prepend anything to the metric name.
```

Comment:
We should perhaps use a pseudo-root in cases only one application is provided and not omit that level in the hierarchy to make things uniform (?)

Reply:
I agree with @pilhuhn. If only one application is deployed, there should be metrics with both context root prefix and without it. That way it wouldn't break the behavior specified in v1.0. 

What happens to the metrics without prefix if another app is deployed would be left unspecified - impls can choose to show the metrics from the app deployed first, last, combine conflicting metrics together or allow user to configure it. I expect that the metrics without a prefix would be only used when there's a single app deployed.